test/lib/linter/rules/snapshots/NoDeprecatedApi.ts.md

```diff
@@ -54,15 +54,130 @@ Generated by [AVA](https://avajs.dev).
             severity: 2,
           },
           {
-            column: 24,
+            column: 3,
             line: 16,
+            message: 'Bootstrap parameter \'data-sap-ui-animation\' should be replaced with data-sap-ui-animation-mode',
+            messageDetails: 'Migrate to \'data-sap-ui-animation-mode\' attribute AnimationMode (https://ui5.sap.com/1.120/#/api/module:sap/ui/core/AnimationMode)',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 17,
+            message: 'Abandoned bootstrap parameter \'data-sap-ui-areas\' should be removed',
+            ruleId: 'no-deprecated-api',
+            severity: 1,
+          },
+          {
+            column: 3,
+            line: 18,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-auto-aria-body-role\' should be removed',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 19,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-binding-syntax\' should be removed',
```

Comment:
Also here. Why can/should it be removed?

Reply:
Done